### PR TITLE
Add Symfony 6 support

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

Part of: https://github.com/sonata-project/dev-kit/issues/1675
Dev-kit PR: https://github.com/sonata-project/dev-kit/pull/1781

## Changelog

```markdown
### Added
- Added support for Symfony 6
```

### CODE REVIEW

**Comment by jordisala1991** [comment]
Blocked by `doctrine/mongodb-odm-bundle`

**Comment by VincentLanglet** [comment]
> doctrine/mongodb-odm-bundle

Need to follow https://github.com/doctrine/DoctrineMongoDBBundle/pull/710

**Comment by jordisala1991** [comment]
It shouldn't be blocked anymore

**Comment by jordisala1991** [comment]
I might need some help from you @franmomu , there is a single test failing: `EmbeddedMappingTest::testCreateDocumentWithEmbedded`

**Comment by franmomu** [comment]
> I might need some help from you @franmomu , there is a single test failing: `EmbeddedMappingTest::testCreateDocumentWithEmbedded`

Sure, looks like the issue is with the `ResizeFormListener` from `form-extensions`, I'll try to fix it.

File: tests/Functional/EmbeddedMappingTest.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function testCreateDocumentWithEmbedded(): void
         $form[$attributeAddressStreet] = 'A wonderful street to live';
 
         $crawler->filter('.field-container .sonata-ba-action[title="Add new"]')->click();
-        $crawler = $this->client->waitFor('.phone_number_number');
+        // $crawler = $this->client->waitFor('.phone_number_number');
```
Comment (franmomu) [inline tests/Functional/EmbeddedMappingTest.php]: this line can be reverted now and after https://github.com/sonata-project/form-extensions/pull/292 it should work fine

Comment (jordisala1991) [inline tests/Functional/EmbeddedMappingTest.php]:
```suggestion
        $crawler = $this->client->waitFor('.phone_number_number');
```

Comment (jordisala1991) [reply tests/Functional/EmbeddedMappingTest.php]: Done

**Comment by jordisala1991** [comment]
The error looks better now, but it doesn't work still.

**Comment by jordisala1991** [comment]
Finally 🎉

**Review by VincentLanglet — APPROVED** [review]

**Review by core23 — APPROVED** [review]

**Comment by franmomu** [comment]
thanks @jordisala1991!

**Pull request merged**